Src/Base/AMReX_BCUtil.cpp

```diff
@@ -1,6 +1,9 @@
 #include <AMReX_BCUtil.H>
 #include <AMReX_PhysBCFunct.H>
 
+// CUDA 11.6 bug: https://github.com/AMReX-Codes/amrex/issues/2607
+#if !defined(__CUDACC__) || (__CUDACC_VER_MAJOR__ != 11) || (__CUDACC_VER_MINOR__ != 6)
```

Comment:
Do we want to also check the patch version, since @maxpkatz said it will be fixed in the next minor release?

https://github.com/AMReX-Codes/amrex/issues/2607#issuecomment-1076540867

As far as I know, "Update" and minor releases in CTK these days are some patch level change.

Reply:
Definitely, if we know the patch number.

Reply:
@maxpkatz does CUDA 11.6.2 fix this/#2607?
https://github.com/spack/spack/pull/29756

Reply:
Does not seem to be the case.  I was using 11.6.1 for this PR.  I also just tested 11.6.2.  Still saw the same "invalid device function" error.